frontend/tests/unit/components/note/NoteBanner.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { mount } from '@vue/test-utils';
+
+import NoteBanner from '@/components/note/NoteBanner.vue';
+import { StorageKey } from '@/utils/enums/application';
+import PrimeVue from 'primevue/config';
+import Tooltip from 'primevue/tooltip';
+
+import type { Note } from '@/types';
+
+vi.mock('vue-i18n', () => ({
+  useI18n: () => ({
+    t: vi.fn()
+  })
+}));
+
+const currentDate = new Date().toISOString();
+
+const testNote: Note = {
+  noteId: 'noteUUID',
+  note: 'This is a test note.',
+  createdBy: 'testCreatedBy',
+  createdAt: currentDate,
+  updatedBy: 'testUpdatedBy',
+  updatedAt: currentDate
+};
+
+const wrapperSettings = (testNoteProp = testNote) => ({
+  props: {
+    note: testNoteProp
+  },
+  global: {
+    plugins: [() => PrimeVue],
+    stubs: ['font-awesome-icon'],
+    directives: {
+      Tooltip: Tooltip
+    }
+  }
+});
+
+beforeEach(() => {
+  sessionStorage.setItem(
+    StorageKey.CONFIG,
+    JSON.stringify({
+      oidc: {
+        authority: 'abc',
+        clientId: '123'
+      }
+    })
+  );
+
+  vi.clearAllMocks();
+});
+
+afterEach(() => {
+  sessionStorage.clear();
+});
+
+describe('NoteBanner', () => {
+  it('renders component', async () => {
+    const wrapper = mount(NoteBanner, wrapperSettings());
+    expect(wrapper).toBeTruthy();
+  });
+});
```

Comment:
Since the wrapperSettings function is able to control what prop is being jammed in as a note, we should at minimum be testing that the jammed in string appears in the DOM. A truthy render test doesn't actually provide much value other than coverage and a guarantee that it is renderable.